### PR TITLE
Update Default Username in Manage the Root User on Astronomer Software

### LINKED ISSUE #1803: Astronomer Software root username is incorrect
**Affected Document:**
https://docs.astronomer.io/software/manage-root-user
**Problem Type (Priority):**
This part of the documentation did not work for me. I kept getting invalid username or password errors. I managed to connect with  `root` instead of `root@<your-basedomain>`.

> When you install Astronomer Software, a root user with all permissions is automatically created. The root user's username is root@<your-basedomain>, and the password is stored as a Kubernetes secret on your installation.

- [X] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [ ] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

> When you install Astronomer Software, a root user with all permissions is automatically created. The root user's username is `root`, and the password is stored as a Kubernetes secret on your installation.

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1803

### CODE REVIEW

**Comment by kwcanuck** [comment]
Is this still an option on the Astronomer Software log in page:

image

I didn't see this option in my instance.

**Review by odaneau-astro — APPROVED** [review]

**Comment by odaneau-astro** [comment]
> Is this still an option on the Astronomer Software log in page:
> 
> image
> 
> I didn't see this option in my instance.

Same here. The button text is the same as with any other user logging in.

**Review by odaneau-astro — APPROVED** [review]

File: software/manage-root-user.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
 ---
-title: 'Manage the root user on Astronomer Software'
+title: 'Manage the Astronomer Software root user'
 sidebar_label: 'Manage the root user'
 id: manage-root-user
 description: Learn how to manage the root user role with all permissions on Astronomer Software
 ---
 
-When you install Astronomer Software, a root user with all permissions is automatically created. The root user's username is `root@<your-basedomain>`, and the password is stored as a Kubernetes secret on your installation.
+When you install Astronomer Software, a root user with the username `root` and all permissions is automatically created. The password for the `root` user is stored as a Kubernetes secret on your Astronomer Software installation.
```
Comment (jwitz) [inline software/manage-root-user.md:8]:
```suggestion
When you install Astronomer Software, a root user with the username `root` and all permissions is automatically created. The password for the root user is stored as a Kubernetes secret on your Astronomer Software installation.
```

File: software/manage-root-user.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
-kubectl get secret astronomer-root-admin-credentials -o jsonpath='{.data.password}' -n <your-platform-namespace> | base64 --decode
-```
-
-To log in to Astronomer Software as the root user, retrieve the password and click **Root Admin Login** on the Software login page.
+2. Log in to your Astronomer Software instance with the root username and password.
 
 ## Customize the root user's password
 
-You might want to use a custom root user password to ensure that someone who has recently left your organization no longer has access to the root user. 
+You can create a custom root user password to limit access to the root user account.
```
Comment (jwitz) [inline software/manage-root-user.md:22]:
```suggestion
You might want to use a custom root user password to ensure that someone who has recently left your organization no longer has access to the root user.
```
The original wording is more precise. The reason you would want to change the password is to cut somebody off who previously had access. Changing it without this context wouldn't limit access in and of itself.

**Review by jwitz — APPROVED** [review]
LGTM! Just a few small suggestions to bring back some context around the feature

**Pull request merged**